components/openindiana/bhyve-fw/bhyve-fw.p5m

```diff
@@ -0,0 +1,30 @@
+#
```

Comment:
Can you replace bhyve-fw.p5m with sample-manifest (adding your copyright there)?

Reply:
The sample one is missing the directory entries and the links, as well as the 555 permissions on the files themselves. Can you let me know what I need to do?

Reply:
1) dir actions are not compulsory, if directories are not empty.
2) links should be added separately, before auto-generated section from sample-manifest (Have I pointed to https://github.com/OpenIndiana/oi-userland/blob/oi/hipster/doc/best-practices.md ?)
3) mode is better set with transform (or in individual action if you prefer).